Objective-C/Tests/MessageEndpointTest.m

```diff
@@ -174,7 +174,9 @@ - (void) startDiscovery {
     _browser.delegate = self;
     [_browser startBrowsingForPeers];
     
-    [self waitForExpectations: @[_clientConnected, _serverConnected] timeout: 10.0];
+    [self ignoreException:^{
+        [self waitForExpectations: @[_clientConnected, _serverConnected] timeout: 10.0];
```

Comment:
Does waitForExpectations throw connection reset exception? It doesn't seem like to. Is it the correct place to ignore the exception?